pom.xml

```diff
@@ -11,25 +11,25 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <segue.version>v3.16.3-SNAPSHOT</segue.version>
-        <log4j.version>2.22.1</log4j.version>
-        <resteasy.version>6.2.8.Final</resteasy.version>
+        <log4j.version>2.23.1</log4j.version>
+        <resteasy.version>6.2.9.Final</resteasy.version>
         <guice.version>7.0.0</guice.version>
-        <oauth-client.version>1.35.0</oauth-client.version>
-        <jackson.version>2.17.0</jackson.version>
+        <oauth-client.version>1.36.0</oauth-client.version>
+        <jackson.version>2.17.2</jackson.version>
         <jackson-databind.version>2.17.0</jackson-databind.version>
         <powermock.version>2.0.9</powermock.version>
         <junit-4.version>4.13.2</junit-4.version>
-        <junit-5.version>5.10.2</junit-5.version>
+        <junit-5.version>5.10.3</junit-5.version>
         <swagger-ui-version>4.13.2</swagger-ui-version>
         <prometheus.version>0.16.0</prometheus.version>
-        <jetty-version>11.0.20</jetty-version>
+        <jetty-version>11.0.22</jetty-version>
```

Comment:
This change needs a corresponding change to the two Dockerfiles: `Dockerfile-api` and `Dockerfile-etl` otherwise things might break when deployed.